test/NCI.OCPL.Api.Glossary.Tests/TestData/TermsControllerData/TestData_GetWithFallback_GeneticsHP.json

```diff
@@ -0,0 +1 @@
+{"TermId":556486,"Language":"en","Dictionary":"Genetics","Audience":"HealthProfessional","TermName":"deleterious mutation","FirstLetter":"d","PrettyUrlName":"deleterious-mutation","Pronunciation":{"Key":"(DEH-leh-TEER-ee-us myoo-TAY-shun)","Audio":"https://nci-media-dev.cancer.gov/pdq/media/audio/736913.mp3"},"Definition":{"Html":"A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain.  Also called disease-causing mutation, pathogenic variant, predisposing mutation,  and susceptibility gene mutation.","Text":"A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain.  Also called disease-causing mutation, pathogenic variant, predisposing mutation,  and susceptibility gene mutation."},"RelatedResources":[],"Media":[]}
```

Comment:
This is going to be a pain to work with if it ever needs to be updated.   Will the test work with the data in a human-friendly format?

Reply:
Answering my own question...  The tests, as-is, will break if the JSON is formatted to be human-readable.

However, this will be painful to maintain, and fragile too as string comparison doesn't allow for properties to be in a different order.

In the test class, use `JObject.Parse()` to convert the strings to JObject structures.  (A helper function could wrap the `File.ReadAllText()`,   Potentially, there could be a `DeserializeJson` method in `TestingTools`.)

Then, instead of comparing two strings, 
```csharp
Assert.Equal(expectedData, actualData, new JTokenEqualityComparer());
```

This would allow both the data files to be human-readable and properties to be in a different order.  (Array contents are expected to remain in the same order regardless.)